src/Inputs/Checkboxes.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Jeremeamia\Slack\BlockKit\Inputs;
+
+use Jeremeamia\Slack\BlockKit\Exception;
+use Jeremeamia\Slack\BlockKit\Partials\Option;
+
+class Checkboxes extends InputElement
+{
+
+    use HasConfirm;
+
+    private const MIN_OPTIONS = 1;
+    private const MAX_OPTIONS = 10;
+
+    /** @var array|Option[] */
+    private $options = [];
+
+    /** @var array|Option[] */
+    private $initialOptions = [];
+
+    public function addOption(Option $option, bool $isInitial = false): Checkboxes {
+
+        $this->options[] = $option;
+
+        if ($isInitial) {
+            $this->initialOptions[] = $option;
+        }
+
+        return $this;
+
+    }
+
+    public function validate(): void
+    {
+
+        if (count($this->options) > self::MAX_OPTIONS) {
+            throw new Exception('Option Size cannot exceed %d', [self::MAX_OPTIONS]);
+        }
+
+        if (count($this->options) < self::MIN_OPTIONS) {
+            throw new Exception('Option Size must be at least %d', [self::MIN_OPTIONS]);
+        }
+
+    }
+
+    /**
+     * @return array
+     */
+    public function toArray(): array
+    {
+        $data = parent::toArray();
+
+        if (! empty($this->initialOptions)) {
+            $data['initial_options'] = $this->initialOptions;
```

Comment:
Should be:
```
$data['initial_options'] = array_map(function (Option $option) {
    return $option->toArray();
}, $this->initialOptions);
```

Reply:
i thought the conversion will be done implicitly, it actually works without array_map, are you sure it is needed here ?

Reply:
It's not necessarily needed for the JSON conversation, but currently every other element's toArray method is recursive, such that toArray's output contains only scalars with the arrays.